notebooks/06_Serving/06_01_Seldon_Core_Serving.ipynb

```diff
@@ -0,0 +1,185 @@
+    "\n",
+    "Seldon Core is an open source platform to deploy your machine learning models on Kubernetes at massive scale.It converts ML models (Tensorflow, Pytorch, H2o, etc.) or language wrappers (Python, Java, etc.) into production REST/GRPC microservices.\n",
+    "\n",
+    "Seldon handles scaling to thousands of production machine learning models and provides advanced ML capabilities out of the box including Advanced Metrics, Request Logging, Explainers, Outlier Detectors, A/B Tests, Canaries and more."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Note: \n",
+    "Since you're in anonymous namespace lack of permission to create other namespaces and perform operations on that namespace, you should copy all the yaml files and execute all of the below code blocks in your local environment. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 1. Create New Namespace"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!kubectl create ns testseldon"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 2. Label that namespace so you can run inference tasks in it"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!kubectl label namespace testseldon serving.kubeflow.org/inferenceservice=enabled"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 3. Create an Istio gateway in that namespace named `kubeflow-gateway`"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!kubectl create -f gateway.yaml"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 4. Create an example SeldonDeployment with a dummy model"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!kubectl create -f seldondeployment.yaml"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 5.Wait for state to become available"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!kubectl get sdep seldon-model -n testseldon -o jsonpath='{.status.state}\\n'"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 6. Port forward to the Istio gateway"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!kubectl port-forward $(kubectl get pods -l istio=ingressgateway -n istio-system -o jsonpath='{.items[0].metadata.name}') -n istio-system 8004:80"
```

Comment:
You do use `istio-ingressgateway`, why do you create `kfserving-gatway` at the beginning?

Reply:
I haven't created a kfserving-gateway, it's still an istio-ingressgateway. But we don't have to do it now since we can just create resources in kubeflow namespace.